aws-dual-stack/README.md

```diff
@@ -0,0 +1,68 @@
+# How to create a new dual-stack cluster in aws
+
+### Prepare for installation
+Download openshift installer to the _install_ directory
+https://developers.redhat.com/products/openshift/download
+- "Install Red Hat OpenShift on your laptop"
+- click "Downloads" on left menu 
+- Download "OpenShift command-line interface (oc)"
+- Download "OpenShift for x86_64 Installer"
+- Scroll to bottom "Tokens" Download Pull Secret _install_/pull-secret.txt
+
+Create a new ssh key in the _install_ directory
+`ssh-keygen -t rsa -f _install_/openshift-test-ssh`
+
+Edit params at the top of `./openshift-cluster.sh`
+
+### Create the AWS OpenShift Cluster
+
+```bash
+./openshift-cluster.sh test-openshift-5
+```
+This shell script will create a full working OpenShift 4.x cluster in AWS. Successful run will output 'INFO Install complete!'
```

Comment:
Let's test the latest version in case openshfit 4.11 has dual-stack support

Reply:
Let's just link to OpenShift AWS install instructions to say what is needed to setup your own cluster. Those same privileges exist for the dual-stack ones.